src/layout/flex-grid/src/index.tsx

```diff
@@ -107,7 +82,13 @@ export const Row: React.FC<RowProps> = ({
       sx={{
         variant: `grid.row`,
         mx: (theme: any): any =>
-          getNegativeSpaceValue(getGutterSize(theme))(theme),
+          removeArrayIfSingleItem(
+            alwaysGetArray(getGutterSize(theme))
+              .map((value: any) => getValueFromSize(theme, value))
```

Comment:
I don't think you need these types here as typescript should get them from the function signatures, right?

(not that it hurts or really makes any difference at all)

Reply:
it whines without so leaving